src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu-divider/responsive-vertical-menu-divider.style.ts

```diff
@@ -7,15 +7,8 @@ interface StyledResponsiveVerticalMenuDividerProps {
 }
 
 export const StyledResponsiveVerticalMenuDivider = styled.div<StyledResponsiveVerticalMenuDividerProps>`
-  width: 100%;
-
-  ${({ depth, responsive }) =>
-    depth &&
-    depth > 0 &&
-    responsive &&
-    css`
-      width: 88%;
-    `}
+  width: ${({ depth, responsive }) =>
+    depth && depth > 0 && responsive ? "88%" : "100%"};
```

Comment:
question: won't this always be defined (I could be wrong)? If so it can just be `depth > 0`